charts/public-service/templates/ingress.yaml

```diff
@@ -35,4 +35,13 @@ spec:
               serviceName: {{ $fullName }}
               servicePort: service-port
   {{- end }}
+  {{- if not  .Values.ingress.hosts }}
```

Comment:
As this is a `not` statement, I guess this deserves a bit of documentation somewhere 😄

Reply:
d919f2d